docs/install/index.md

```diff
@@ -3,36 +3,29 @@ html_meta:
   "description": "Install Plone 6"
   "property=og:description": "Install Plone 6"
   "property=og:title": "Install Plone 6"
-  "keywords": "Plone 6, install, installation, installing, buildout, pip, docker"
+  "keywords": "Plone 6, install, overview"
 ---
 
 (install-index-label)=
 
 # Install Plone 6
 
-This section focuses on installing Plone 6 for the developer, tester, or contributor.
 
-For trying out Plone 6, visit our official demonstration site located at:
+(install-index-getting-started-label)=
 
-```{toctree}
-:maxdepth: 2
-:hidden: true
+## Getting started
 
-containers/index
-```
+* What do you want to do?
+  * Try a demo now
```

Comment:
Links to both demos:
https://6.demo.plone.org/
https://demo.plone.org/

Reply:
Added https://6.demo.plone.org/ in ce27a9f8e48aab6bdc2719709fb3cdbc299fa209

https://demo.plone.org/ is 404.

Reply:
https://demo.plone.org/  should not be 404. But of course this does not need to be mentioned.